tags/dataset.yaml

```diff
@@ -85,6 +85,8 @@ values:
     display_name: Kinetics-400
   - id: kinetics-600
     display_name: Kinetics-600
+  - id: librispeech
```

Comment:
We've already added the dataset for you via 309f90d36e658e69ac970de70729ea7760c2f76a so feel free to merge the current master into this branch :)

Reply:
forked master to this patch-1.right?

Reply:
Yes I think something like this should work:
```
git checkout patch-1
git pull origin master
```

Reply:
Done.